src/components/list/ListPage.js

```diff
@@ -30,6 +33,20 @@ const ListPage = () => {
     }
   }, [dispatch, locationsAvailable, locationsInvalid])
 
+  useEffect(() => {
+    if (lastViewedListPositionId) {
+      const locationElement = document.getElementById(lastViewedListPositionId)
+      if (locationElement) {
+        const scrollParent = locationElement.parentElement
+        if (scrollParent) {
```

Comment:
This will do if we can't make scrollIntoView work but we're losing some intelligent behaviour it has. Can you try modifying the styling of top component or putting a div around each location entry? I can have a go as well

Reply:
I tried different approaches with `scrollIntoView()` but all of them caused the scroll container to shift slightly behind the header. The manual calculation with `offsetTop` and `scrollTop` is the only approach that worked. Feel free to give it a try if you have other ideas.